src/controller/audio-track-controller.js

```diff
@@ -108,8 +115,6 @@ class AudioTrackController extends TaskLoop {
 
     logger.log(`audioTrack ${data.id} loaded`);
 
-    this.tracks[data.id].details = data.details;
```

Comment:
@gkolb Where do you set the details if not here?

Reply:
@realeyes-matthew In the onTrackLoaded in the audio-stream-controller.js, this was overwriting the merge details for live streams with the new details and it was causing bugs

Reply:
Sounds good